src/utils/ICS20Lib.sol

```diff
@@ -75,16 +75,16 @@ library ICS20Lib {
         returns (bytes memory)
     {
         return abi.encodePacked(
-            "{\"amount\":\"",
-            Strings.toString(amount),
-            "\",\"denom\":\"",
+            "{\"denom\":\"",
```

Comment:
These were changed to match the order they come in from ibc-go. This is a limitation of the ICS20Lib: it only supports JSON in a specific order. Parsing the JSON in any order is possible, but might be more gas-intensive. Should we create an issue for that? There is a TODO below on this.

Reply:
Yes, let's create an issue. And please link the issues to the todo items (as numbers) if there is an associated issue.